libs/blocks/marquee/mnemonic-list.js

```diff
@@ -0,0 +1,26 @@
+import { getConfig, createTag, loadStyle } from '../../utils/utils.js';
+
+export const decorateMnemonicList = (container) => {
+  if (!container) return;
+  const paragraphs = container.querySelectorAll(':scope p:not([class])');
+  if (paragraphs.length < 1) return;
+  const { miloLibs, codeRoot } = getConfig();
+  loadStyle(`${miloLibs || codeRoot}/blocks/marquee/mnemonic-list.css`);
```

Comment:
I would move this to marquee.js so that it gets loaded faster.

Reply:
Great suggestion, it's made.